ui/src/pages/Deployment.vue

```diff
@@ -72,17 +67,17 @@
     interval: 300000,
   }
 
+  const deploymentSubscription = useSubscription(api.deployments.getDeployment, [deploymentId.value], subscriptionOptions)
+  const deployment = computed(() => deploymentSubscription.response)
```

Comment:
Technically we were trying to access these before initialization in `computedTabs`